src/cloud-api-adaptor/go.mod

```diff
@@ -14,8 +14,8 @@ require (
 	github.com/aws/aws-sdk-go-v2/config v1.15.11
```

Comment:
Now I found it :)

I believe you could split this PR and send this bump in its own PR to unlock our CI. This is a big PR, l and I'm already missing this bump itself.